jenkins/job-configs/kubernetes-jenkins/bootstrap-ci.yaml

```diff
@@ -192,6 +192,12 @@
         frequency: 'H/5 * * * *' # At least every 30m
         json: 1
         trigger-job: 'kubernetes-build'
+    - kubernetes-e2e-gce-taint-evict:
+        job-name: ci-kubernetes-e2e-gce-taint-evict
+        timeout: 80
```

Comment:
https://github.com/kubernetes/test-infra/pull/2008 renamed `timeout` to `jenkins-timeout`